packages/api/lambdas/write-db-dlq-records-to-s3.js

```diff
@@ -31,7 +33,7 @@ async function handler(event) {
     await s3PutObject({
       Bucket: process.env.system_bucket,
       Key: `${process.env.stackName}/dead-letter-archive/sqs/${s3Identifier}.json`,
-      Body: sqsMessage.body,
+      Body: JSON.stringify(sqsMessage),
```

Comment:
Would you mind also adding the following?
```
ContentType: 'application/json'
```
Without this, AWS defaults to `application/octet-stream`, which prevents users in the AWS Console from being able to open JSON files in the browser. Instead, the browser forces the user to download the file, which becomes rather annoying.

This is true anywhere in the Cumulus codebase that writes JSON files (such as CMA messages written to S3).

Perhaps I should create a Cumulus issue requesting that `ContentType` be added to every call to `s3PutObject`?